app/src/main/java/com/khaithu/a3ctech_android_final_project/presenter/UserPresenter.kt

```diff
@@ -0,0 +1,62 @@
+package com.khaithu.a3ctech_android_final_project.presenter
+
+import com.google.firebase.auth.FirebaseAuth
+import com.google.firebase.database.*
+import com.khaithu.a3ctech_android_final_project.model.WatchlistItem
+import com.khaithu.a3ctech_android_final_project.presenter.interfacePre.IUserPresenter
+import com.khaithu.a3ctech_android_final_project.view.interfaceView.IUserLoggedInView
+
+class UserPresenter(var mIUserLoggedInView: IUserLoggedInView) : IUserPresenter {
+
+    private lateinit var rootNode: FirebaseDatabase
+    private lateinit var reference: DatabaseReference
+    private lateinit var firebaseAuth: FirebaseAuth
+
+    override fun getTitle() {
+        firebaseAuth = FirebaseAuth.getInstance()
+        rootNode = FirebaseDatabase.getInstance()
+        reference = rootNode.reference
+
+        firebaseAuth.currentUser?.uid?.let {
+            reference.child("User").child(it).child("fullName").get()
+                .addOnSuccessListener { task ->
+                     mIUserLoggedInView.updateTitle(task.value.toString())
+                }
+        }
+    }
+
+    override fun createWatchlist() {
+        firebaseAuth = FirebaseAuth.getInstance()
+        rootNode = FirebaseDatabase.getInstance()
+        reference = rootNode.reference
+
+        firebaseAuth.currentUser?.uid?.let {
+            reference.child("User").child(it).child("Watchlist")
```

Comment:
you can define some firebase keys such as "User", "Watchlist" at new class `FirebaseConstant`